src/main/java/org/semver4j/internal/range/processor/IvyProcessor.java

```diff
@@ -29,10 +31,12 @@
  * </ul>
  */
 public class IvyProcessor implements Processor {
+    @NotNull
```

Comment:
That's not necessary in favor of field object assignment.

Reply:
Reasons 1 & 6